index.html

```diff
@@ -31,9 +33,42 @@ <h1>Welcome to the student showcase!</h1>
         <div>ELEVEN</div>
         <div>TWELVE</div>
         <div>THIRTEEN</div>
-        <div>FOURTEEN</div>
+        <div>
+            <div id="header">
+                <h1>Nicholas McVickar</h1>
+            </div>
+            <div id="content">
+                <h2>"A bird does not sing because it has an answer, it sings because it has a song."</h2> <br>
```

Comment:
You won't need that `<br>`. We'll use `margin`/`padding` instead to space these `h2` further apart